### DISCUSSION TITLE
Lost the AIO setup password, what can I do?

### DISCUSSION THREAD

**Discussion by plrenaudin** [nextcloud/all-in-one#803] [Questions] [answered]
Just rebooted my server and Nextcloud requires the setup password that somehow I lost... 
Is there anything I can do to reset it?
![image](https://user-images.githubusercontent.com/10697451/172594946-1393157f-b2cf-4ba2-986b-007c02eeb8e5.png)

Thanks

**Comment by szaimen** [comment, marked as answer]
Hi, you should be able to get the mastercontainer password by running
`sudo cat /var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json | grep password`
(It is the value of `password`).

**Reply by plrenaudin** [reply]
Thanks, that worked!

**Reply by BigRrred** [reply]
Thank you so much! It worked! Was worried I was stuck with the same issue.

**Reply by TheNomad11** [reply]
Thanks! I am relieved. Maybe you could add this to the FAQ in the Readme?

**Reply by szaimen** [reply]
I've created https://github.com/nextcloud/all-in-one/discussions/1786

**Reply by v10o** [reply]
That just worked!!!

**Reply by diyoyo** [reply]
Just adding an alternative if this does not work:
`docker exec -it nextcloud-aio-mastercontainer cat /mnt/docker-aio-config/data/configuration.json | grep password`

**Reply by bogganj94** [reply]
your my hero take a heart emoji

**Reply by ArmanTheParman** [reply]
I tried this but the file doesn't exist. The dir does but not the file, just an empty dir

**Comment by steliosap17** [comment]
what if i use windows?

**Reply by AdnanSattar** [reply]
its here from file Explorer 
\\wsl.localhost\docker-desktop\mnt\docker-desktop-disk\data\docker\volumes\nextcloud_aio_mastercontainer\_data\data
